Userland/Libraries/LibCore/EventLoop.h

```diff
@@ -141,6 +146,7 @@ class EventLoop {
     };
 
     Vector<QueuedEvent, 64> m_queued_events;
+    NonnullRefPtrVector<Promise<NonnullRefPtr<Object>>> m_running_jobs;
```

Comment:
This name of "running jobs" doesn't really explain that it's going to be a collection of promises. `m_pending_promises` would be clearer.

Reply:
That would probably require redesigning a lot of  users